go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ONSdigital/dp-api-clients-go v1.31.0
 	github.com/ONSdigital/dp-graph/v2 v2.3.0
 	github.com/ONSdigital/dp-healthcheck v1.0.5
-	github.com/ONSdigital/dp-kafka v1.1.7
+	github.com/ONSdigital/dp-kafka/v2 v2.1.1
```

Comment:
can we use `dp-kafka v2.1.2` please? (I removed a log that was too noisy and made debugging of issues more difficult)